### ISSUE TITLE
adding storage class breaks the cgi server

### ISSUE THREAD

**Issue by dragon2611** [moosefs/moosefs#186]
Doesnt seem to matter what I call the storage class (e.g even if I just use 10) it blows up the resources page of the CGI server
> Traceback (most recent call last):
>   File "/usr/share/mfscgi/mfs.cgi", line 5989, in 
>     color = "#%02X0000" % (255-255*msperc)
> TypeError: %X format: an integer is required, not float

Remove the storage class and it's happy again.

version: 3.0.103-1 - Ubuntu 18.04

**Comment by onlyjob** [comment]
Definitely there is a problem, at least with Python3. Not only adding a new storage class breaks "Resources" tab in CGI but modifying a class yields the same effect.

To fix the problem I've applied the following change to lines 5989, 6007, 6025:

~~~~
-  color = "#%02X0000" % (255-255*msperc)
+  color = "#%02X0000" % int(255-255*msperc)
~~~~

Please review if that's correct as Python is not my lingo...

**Comment by acid-maker** [comment]
Yes I know about that. It has been fixed month ago. I need to update sources on github.

**Comment by acid-maker** [comment]
It is even already fixed by commit e21874a47b4db9d838fa69ddc106c1345f9b31be
